docs/codeql/CONTRIBUTING.MD

```diff
@@ -0,0 +1,18 @@
+# Contributing to CodeQL docs
```

Comment:
Minor: Typically, the file extension is lower case `docs/codeql/CONTRIBUTING.MD` -> `docs/codeql/CONTRIBUTING.md`